spec/acceptance/nodesets/docker_alma8.yml

```diff
@@ -0,0 +1,18 @@
+HOSTS:
+  alma8:
+    roles:
+      - client
+    platform: el-8-x86_64
+    hypervisor: docker
+    image: almalinux:8
+    docker_cmd: '/usr/sbin/init'
+    docker_preserve_image: true
+    docker_image_commands:
+      # Puppet Deps
+      - 'yum install -y rsync openssl openssh-server'
+CONFIG:
+  log_level: verbose
+  type: aio
+<% if ENV['BEAKER_PUPPET_COLLECTION'] -%>
+  puppet_collection: <%= ENV['BEAKER_PUPPET_COLLECTION'] %>
+<% end -%>
```

Comment:
Just so these will pass `yamllint`, could you make that
```yaml
  puppet_collection: <= ENV.fetch('BEAKER_PUPPET_COLLECTION', '~') %>
```
Alternatively, we *could* hard-code that to `openvox8` instead of `~` now, I think.